rclpy/rclpy/qos.py

```diff
@@ -270,8 +270,11 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#  Naming the class HistoryPolicy QoSHistoryPolicy 
+#  because it is used in other places with the "Qos" infront
+QoSHistoryPolicy=HistoryPolicy
```

Comment:
I think the whitespace on this line will fail the linter test. It should be `QoSHistoryPolicy=HistoryPolicy` instead, and same a few places below.